lib/features/add_property/view_model/add_property_images/add_property_images_cubit.dart

```diff
@@ -24,11 +24,7 @@ class AddPropertyImagesCubit extends Cubit<AddPropertyImagesState> {
   }
 
   void onSelectPhotoBottomSheetDismissed(List<String>? result) {
```

Comment:
Should we remove **List<String>? result** ?>

Reply:
Yeah, let me remove it